.github/workflows/liblab-sdk-updates.yaml

```diff
@@ -11,7 +11,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+      - name: Verify liblab token
+        if: ${{ env.LIBLAB_TOKEN == '' }}
+        run: echo "::warning::LIBLAB_TOKEN has not been set"
+      - name: Verify github token
+        if: ${{ env.LIBLAB_GITHUB_TOKEN == '' }}
+        run: echo "::warning::LIBLAB_GITHUB_TOKEN has not been set"
```

Comment:
Not needed here, it's already included as a check in Github Action implementation.